### ISSUE TITLE
reproduction in keras doesn't give same accuracy

### ISSUE THREAD

**Issue by asadisina** [ARM-software/ML-KWS-for-MCU#148]
Hi,

I have been trying to reproduce the same DS-CNN architecture in Keras mode in Edge Impulse Studio. I implemented the exact same arch and try to give the system the same dataset as training and validation. but I see an overfitting issue (training accuracy goes to 100 but validation accuracy stays around 88 percent).
Do you have any idea why reproducing in Keras giving me this overfitting issue?

Thank you in advance,

**Comment by Burton2000** [comment]
I know its been a long time since this question was asked but incase it is useful to anyone else, if you go to here: https://github.com/ARM-software/ML-examples/tree/main/tflu-kws-cortex-m/Training we have implemented training of the networks using TF2 and Keras APIs. Accuracy is maintained so there is nothing about swapping to Keras that should reduce model accuracy
